src/commcare_cloud/manage_commcare_cloud/make_docs.py

```diff
@@ -70,7 +70,7 @@ def wrap_lines(text):
     def escape_html_outside_backticks(text):
         parts = text.split('`')
         for i in range(0, len(parts), 2):
-            parts[i] = cgi.escape(parts[i])
+            parts[i] = html.escape(parts[i], quote=False)
```

Comment:
cgi.escape is deprecated in 3.2 and removed in 3.8. The alternative is `html.escape` which has been included since 3.2.